### ISSUE TITLE
Link type "Category" - is empty

### ISSUE THREAD

**Issue by ManiakMill** [verbb/hyper#211]
### Describe the bug

I added custom categories and try to choose by field.  All categories have items.

<img width="753" alt="{0BA33F25-6C45-4E47-A313-4B8EF493E64A}" src="https://github.com/user-attachments/assets/b24022ca-15cf-45f5-bb32-5bad1ff242a7">

But overview is empty:
<img width="1269" alt="{DCA77139-08E2-4873-8527-A69A8B16AEED}" src="https://github.com/user-attachments/assets/c4333813-be93-4a99-bb63-6d69f95881cb">

### Steps to reproduce

1. 
2. 

### Craft CMS version

Craft Pro 5.4.10.1

### Plugin version

2.1.2

### Multi-site?

No

**Comment by ManiakMill** [comment]
Its happened when Category URI Format is empty

**Comment by engram-design** [comment]
That's correct, Hyper will only show elements that have a URL by default. This was a feature request from many of the other link plugins which we incorporated into Hyper. 

But it's also a reasonably common issue that crops up so I'm thinking I might need to add a plugin setting to control this.

The logic here is that there's not much use selecting an element for a link, if that element _has_ no URL - the thing that makes a link. But then, there are always exceptions like User elements, which don't have a URL. But you might be constructing that logic in your templates manually, and you'd still want your content editors able to pick a User element.

**Comment by engram-design** [comment]
Updated in [3.0.0-beta.1](https://github.com/verbb/hyper/releases/tag/3.0.0-beta.1).
